test/models.rb

```diff
@@ -3,6 +3,7 @@ class WithGlobalUID < ActiveRecord::Base
 end
 
 class WithoutGlobalUID < ActiveRecord::Base
+  disable_global_uid
```

Comment:
I guess some tests have been testing the wrong thing until now?

Reply:
Prior to this we were only testing the `MigrationExtension` with it disabled and the impact it had on the generated tables, not the model.